x/budget/spec/05_events.md

```diff
@@ -2,9 +2,11 @@
 
 # Events
 
+The budget module emits the following events.
+
 ## BeginBlocker
 
-### Budget Collection Result for each Budget on this block
+### Budget Collection Result for Each Budget on This Block
```

Comment:
Do we need to use `T` as capital letter in `This`?

Reply:
good question! In this case, yes, it's correct to capitalize "This" I sometimes wonder too and recommend using 
https://capitalizemytitle.com/
![image](https://user-images.githubusercontent.com/10067215/144854345-b6033bdf-076d-42a1-8201-8217569e004f.png)